src/Types/Type.php

```diff
@@ -85,10 +85,7 @@ public function convertToDatabaseValue(mixed $value, AbstractPlatform $platform)
      *
      * @throws ConversionException
      */
-    public function convertToPHPValue(mixed $value, AbstractPlatform $platform): mixed
-    {
-        return $value;
-    }
+    abstract public function convertToPHPValue(mixed $value, AbstractPlatform $platform): mixed;
```

Comment:
This is a breaking change. We can't do this in a minor release.

Reply:
So, we use the phpstan annotations for the minor only? and the other solution for a major?